api/config/featured_workspaces_prod.json

```diff
@@ -54,6 +54,12 @@
       "id": "skillsassessmenttrainingnotebooksforusersv7",
       "category": "TUTORIAL_WORKSPACES"
     },
+    {
+      "name": "SAS 101 Data Fundamentals",
+      "namespace": "aou-rw-6d58fdd8",
+      "id": "sas101datafundamentalsv7",
```

Comment:
Where does id come from?

Based on the URL:
https://workbench.researchallofus.org/workspaces/aou-rw-6d58fdd8/sas101datafundamentals/about

It would seem like v7 shouldn't be at the end, but maybe I am misunderstanding something.